src/user-search/scripts/load_accounts.ts

```diff
@@ -0,0 +1,20 @@
+import {logger} from '../../logger'
+import {CachingClient} from '../client'
+import {loadAllAccountNames} from '../indexes'
+
+export function loadThenWriteAccountNames() {
+    const client = new CachingClient()
+    const namesPromise = loadAllAccountNames(client)
+    logger.info(`loading account names`)
+    namesPromise.catch((err) => {
+        logger.error(err)
```

Comment:
should this fail more loudly?

Reply:
This only runs in the Makefile, nonetheless, it probably should